db/seeds.rb

```diff
@@ -0,0 +1,7 @@
+# This file should contain all the record creation needed to seed the database with its default values.
```

Comment:
- [ ] OPTIONAL: Remove the hardcoded "hello", and use this file `seeds.rb` to feed your database with sample data.  

1. Include this code in this file:

```ruby
greeting_atrributes = [
  { text: "hello" },
  { text: "haha" },
  { text: "whats up!" },
  { text: "hey dude!" },
  { text: "nice smile!" }
]
puts "seeding data..."
greeting_atrributes.each do |attributes|
  Message.create(attributes) unless Message.where(attributes).first
end

puts "done!"
```

2. run `rails db:seed`
3. Update your readme with the new setup instructions:  Instruct your users to run rails db:seed after creating the database and adding the migrations.
4. Enjoy your greetings. 💪